Source/MonoGame.Extended/FramesPerSecondCounter.cs

```diff
@@ -4,9 +4,10 @@
 
 namespace MonoGame.Extended
 {
-    public class FramesPerSecondCounter : IUpdate
+    public class FramesPerSecondCounter : DrawableGameComponent
```

Comment:
I get that we need to derive from `DrawableGameComponent` but why remove the `IUpdate` interface?